### ISSUE TITLE
adding key-bindings to page-up/page-down/fast scrolling #enhancement

### ISSUE THREAD

**Issue by moranshuk** [IngoMeyer441/simple-term-menu#31]
i use this beautiful package to list files in massive log folders across multiple endpoints,
and cant jump fast enough interactively through the lists.

will be nice to have a fast scrolling option if page-up/page-down or some common shortcuts like `shift-down/up`

**Comment by moranshuk** [comment]
there's another feature that will be nice to have -> registering functions for specific events, such as:

cursor_move_event -> run a function when ever the highlight cursors moves
on_enter_event -> run a function when the user is has made hes choice

in multi selection:
on_mark_event -> run the function for item marked regardless of the final select/enter press.
(for example: i select a number of items, and i want for each selected item to run a bg thread to download the item to prepare it.)

**Comment by buanzo** [comment]
Hi Ingo,

This would be very useful indeed to support page-up/down. Scrolling through a LONG menu is very tedious. Something I discovered while coding the configuration browser for Mailman3.

**Comment by IngoMeyer441** [comment]
Page up and down key navigation is now implemented in 8a8b4be2ecf978e699df8998315533d5e77b75c3. Could you please test if this suits your needs?

The latest development version can be installed with:

```bash
python3 -m pip install git+https://github.com/IngoMeyer441/simple-term-menu@develop
```

**Comment by buanzo** [comment]
> Page up and down key navigation is now implemented in commit [8a8b4be](https://github.com/IngoMeyer441/simple-term-menu/commit/8a8b4be2ecf978e699df8998315533d5e77b75c3). Could you please test if this suits your needs?

It works wonderfully, and it most certainly suits my needs! Tested a long, long menu (list of configuration values), with and without / filtering.

Thank you. Happy 2023, by the way, Ingo. I'd buy you a beer anytime, thank you for this library!

**Comment by IngoMeyer441** [comment]
Happy new year! [v1.6.0](https://github.com/IngoMeyer441/simple-term-menu/releases/tag/v1.6.0) is released on GitHub and PyPI and contains the fast scrolling feature. Cheers! :beers:
